src/Access/LDAPClient.cpp

```diff
@@ -322,7 +325,7 @@ void LDAPClient::openConnection()
                 final_user_dn = *user_dn_search_results.begin();
             }
 
-            break;
+            return true;
```

Comment:
Actually the end of the function cannot be reached (we either return or throw an exception before it) so I think not having a return statement there might be better because the compiler will immediately tell us if we forget something.

Reply:
Sounds reasonable.